src/main/resources/templates/team_page_apply_team.html

```diff
@@ -16,16 +16,17 @@ <h2  class="team-heading team-heading-current-team" th:text="#{teamPageApply.hea
                     <fieldset>
                         <div class="form-group">
                             <div class="form-group" th:classappend="${#fields.hasErrors('teamName')}? 'has-error'">
-                                <input type="text" class="form-control" id="inputTeam" placeholder="Team Name" name="" th:field="*{teamName}"/>
-                                <span class="help-block" th:if="${#fields.hasErrors('teamName')}" th:errors="*{teamName}">Team name error</span>
+                                <input type="text" class="form-control" id="inputTeam" placeholder="Team Name" th:field="*{teamName}"/>
+                                <span class="account-change-fail" th:if="${#fields.hasErrors('teamName')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{teamName}"></span></span>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)